### PR TITLE
Add analytics data when opening an asset pack or an asset

### PR DESCRIPTION
**Pull request by 4ian**

### CODE REVIEW

File: newIDE/app/src/Utils/Analytics/EventSender.js [hunk 1]
```diff
@@ -352,24 +366,27 @@ export const sendSubscriptionDialogShown = () => {
...
 };
 
-export const sendAssetAddedToProject = ({
-  id,
-  name,
-}: {|
+export const sendAssetAddedToProject = (options: {|
   id: string,
   name: string,
+  kind: 'public' | 'private' | 'unknown',
```
Comment (ClementPasteau) [inline newIDE/app/src/Utils/Analytics/EventSender.js:383]: is there value in adding this prop?
An asset will probably always have the same kind as its pack

File: newIDE/app/src/Utils/Analytics/EventSender.js [hunk 2]
```diff
@@ -289,16 +289,30 @@ export const sendOnboardingManuallyOpened = () => {
   recordEvent('onboarding_manually_opened');
 };
 
-export const sendAssetPackOpened = (assetPackTag: string) => {
-  recordEvent('asset_pack_opened', {
-    assetPackTag,
-  });
+export const sendAssetPackOpened = (options: {|
+  assetPackName: string,
+  assetPackTag?: string,
```
Comment (ClementPasteau) [inline newIDE/app/src/Utils/Analytics/EventSender.js:294]: to be consistent with other functions, change type to `assetPackTag: string | null` ? (same for id)

File: newIDE/app/src/AssetStore/index.js [hunk 3]
```diff
@@ -54,6 +57,27 @@ export type AssetStoreInterface = {|
   onClose: () => void,
 |};
 
+const identifyAssetPackKind = ({
```
Comment (ClementPasteau) [inline newIDE/app/src/AssetStore/index.js:60]: I know PublicAssetPack should be the same as PublicAssetPack, but as it's not, should we say `isPrivate === !!assetPack.content` ?
(this props is only present on private packs)

Comment (ClementPasteau) [reply newIDE/app/src/AssetStore/index.js:60]: This feels convoluted to search in the arrays.
And in the future, if both types merge to have the same props, we could add a `type: private | public` and use it?

**Review by ClementPasteau — APPROVED** [review]
lgtm, few feedbacks/questions

File: newIDE/app/src/AssetStore/index.js (hunk 3 above)
Comment (4ian) [reply newIDE/app/src/AssetStore/index.js:60]: I wonder if it's too fragile, and the day we update the backend to send a content for public packs, we now have wrong analytics. So we broke analytics by adding a field on the backend?

File: newIDE/app/src/Utils/Analytics/EventSender.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/Utils/Analytics/EventSender.js:383]: Probably yes - but this is more for on the posthog/analytics side, to be able to explore multiple events and do a breakdown (i.e: a "group by" in SQL, a "breakdown" in Posthog) using the same field for multiple events.

Comment (4ian) [reply newIDE/app/src/Utils/Analytics/EventSender.js:383]: This being said, I was assuming other events where having this "kind" for asset, which is not the case. I'm removing it!

File: newIDE/app/src/Utils/Analytics/EventSender.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/Utils/Analytics/EventSender.js:294]: Done

File: newIDE/app/src/AssetStore/index.js (hunk 3 above)
Comment (4ian) [reply newIDE/app/src/AssetStore/index.js:60]: I'll go ahead with this and this is embedded in a function, so we an always simplify this later.

**Pull request merged**